app/services/cards_batch_importer.rb

```diff
@@ -0,0 +1,99 @@
+require "pusher"
+require "nokogiri"
+require "open-uri"
+
+class CardsBatchImporter
+  attr_reader :params, :result, :originals, :translations
+
+  def initialize(url:,
+                 original_selector:,
+                 translated_selector:,
+                 user_id:,
+                 block_id:)
+    @params = {
+      url: url,
+      original_selector: original_selector,
+      translated_selector: translated_selector,
+      user_id: user_id,
+      block_id: block_id
+    }
+    @result = {
+      cards_count: 0
+    }
+  end
+
+  def start
+    if selectors_valid?
+      parse
+    else
+      finish("error", "CSS selectors are not valid")
+    end
+  end
+
+  def notify
+    data = prepare_notification_data
+    Pusher.trigger("bg-job-#{ENV["RAILS_ENV"]}-notifier-#{params[:user_id]}", "job_finished", data)
```

Comment:
Line is too long. [99/80]<br>Prefer single-quoted strings inside interpolations.